README.md

```diff
@@ -1,149 +1,43 @@
-| Bit Field | Field Description | Values |
-| --------- | ----------------- | ------ |
-|[31:24]    | Implementer       | 0x41 = ARM |
-|[23:20]    | Variant           | Major revision 0x0  =  Revision 0 |
-|[19:16]    | Architecture      | 0xC  = Baseline Architecture |
-|           |                   | 0xF  = Constant (Mainline Architecture) |
-|[15:4]     | Part Number       | 0xC20 =  Cortex-M0 |
-|           |                   | 0xC60 = Cortex-M0+ |
-|           |                   | 0xC23 = Cortex-M3  |
-|           |                   | 0xC24 = Cortex-M4  |
-|           |                   | 0xC27 = Cortex-M7  |
-|           |                   | 0xD20 = Cortex-M23 |
-|           |                   | 0xD21 = Cortex-M33 |
-|[3:0]      | Revision          | Minor revision: 0x1 = Patch 1 |
-
-
-
-You can view individual examples and additional API information of the statistics collection tools at the bottom of the page in the [related links section](#related-links).
-
-
-### Output
-
-To view the serial output you can use any terminal client of your choosing such as [PuTTY](http://www.putty.org/) or [CoolTerm](http://freeware.the-meiers.org/). Unless otherwise specified, printf defaults to a baud rate of 9600 on Mbed OS.
-
-You can find more information on the Mbed OS configuration tools and serial communication in Mbed OS in the related [related links section](#related-links).
-
-The output should contain the following block transmitted at the blinking LED frequency (actual values may vary depending on your target, build profile, and toolchain):
-
-```
-=============================== SYSTEM INFO  ================================
-Mbed OS Version: 999999
-CPU ID: 0x410fc241
-Compiler ID: 2
-Compiler Version: 60300
-RAM0: Start 0x20000000 Size: 0x30000
-RAM1: Start 0x1fff0000 Size: 0x10000
-ROM0: Start 0x0 Size: 0x100000
-================= CPU STATS =================
-Idle: 98% Usage: 2%
-================ HEAP STATS =================
-Current heap: 1096
-Max heap size: 1096
-================ THREAD STATS ===============
-ID: 0x20001eac
-Name: main_thread
-State: 2
-Priority: 24
-Stack Size: 4096
-Stack Space: 3296
-
-ID: 0x20000f5c
-Name: idle_thread
-State: 1
-Priority: 1
-Stack Size: 512
-Stack Space: 352
-
-ID: 0x20000f18
-Name: timer_thread
-State: 3
-Priority: 40
-Stack Size: 768
-Stack Space: 664
-
-```
 
 ## Troubleshooting
-
 If you have problems, you can review the [documentation](https://os.mbed.com/docs/latest/tutorials/debugging.html) for suggestions on what could be wrong and how to fix it.
 
 ## Related Links
 
 * [Mbed OS Stats API](https://os.mbed.com/docs/latest/apis/mbed-statistics.html)
 * [Mbed OS Configuration](https://os.mbed.com/docs/latest/reference/configuration.html)
 * [Mbed OS Serial Communication](https://os.mbed.com/docs/latest/tutorials/serial-communication.html)
+* [Mbed OS bare metal](https://os.mbed.com/docs/mbed-os/latest/reference/mbed-os-bare-metal.html)
+* [Mbed boards](https://os.mbed.com/platforms/)
```

Comment:
Links above are no longer relevant. 
Instead I would suggest what are the next examples to look at, for example:

- https://github.com/ARMmbed/mbed-os-example-blinky-baremetal
- https://github.com/ARMmbed/mbed-os-example-sys-info
- https://github.com/ARMmbed/mbed-os-example-cpu-usage
- https://github.com/ARMmbed/mbed-os-example-cpu-stats
- https://github.com/ARMmbed/mbed-os-example-thread-statistics

Reply:
I do not think they are irrelevant.
They provide information to modify the application to:

1. see statistics
1. change default parameters of the OS
1. see output on the serial interface
1. reduce the size of the application

It also provide a link to the board a user can use.

The next examples to look at are already provided at the beginning of the README when this [link](https://os.mbed.com/code/) is provided.

Reply:
Fine. I don't have a stromhg opinion on this. Eiter way.